modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v4/form_fields.rb

```diff
@@ -0,0 +1,160 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module PdfConstruction
+    module HigherLevelReview
+      module V4
+        class FormFields
+          # rubocop:disable Naming/VariableNumber
+          # rubocop:disable Layout/LineLength
```

Comment:
I'm not quite sure how to get around disabling these cops. But one option would be creating constants for some of the repeated parts. Like this:

```ruby
  FORM_PATH = 'form1[0].#subform['.freeze
  VETERAN_PATH = "#{FORM_PATH}2].Veterans_".freeze
  CLAIMANT_PATH = "#{FORM_PATH}2].Claimants_".freeze
  ADDRESS_PATH = "#{FORM_PATH}2].CurrentMailingAddress_".freeze
  PHONE_PATH = "#{FORM_PATH}2].Telephone_".freeze
  DATE_PATH = "#{FORM_PATH}4].Date_Signed_".freeze

  FIELD_NAMES = {
    veteran_middle_initial: "#{VETERAN_PATH}Middle_Initial1[0]",
    veteran_ssn_first_three: "#{VETERAN_PATH}SocialSecurityNumber_FirstThreeNumbers[0]",
   ...
    veteran_dob_day: "#{FORM_PATH}2].DOBday[0]",
    veteran_dob_month: "#{FORM_PATH}2].DOBmonth[0]",
   ...
    veteran_state_code: "#{ADDRESS_PATH}StateOrProvince[0]",
   ...
```
That seems a bit cleaner to me. What do you think?

Reply:
IMHO, that would make easily reading the complete name PDF field names(not our names, VA HLR form) a little bit tricky given the multiple levels of indirection......the use case for reading this code most likely would be for a developer attempting to find that field name string within the PDF(i use firefox to look/find the PDF field names within the PDF)